xbmc/addons/addoninfo/AddonInfo.h

```diff
@@ -27,6 +27,12 @@ class CAddonInfo;
 typedef std::shared_ptr<CAddonInfo> AddonInfoPtr;
 typedef std::vector<AddonInfoPtr> AddonInfos;
 
+/*!
+ * Defines the default language to use as fallback, if wanted not available.
```

Comment:
*defines the default language code used as fallback in case the requested language is not available. Used, for instance, to handle content from addon.xml